Description: improve description of configure.ac option --with-extract-dir
Origin: upstream, https://git.ghostscript.com/?p=ghostpdl.git;h=3ab15c3
Author: Julian Smith <julian.smith@artifex.com>
Last-Update: 2021-18-07
---
This patch header follows DEP-3: https://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -2601,7 +2601,9 @@
 # device.
 #
 AC_ARG_WITH([extract-dir],
-    AC_HELP_STRING([--with-extract-dir=EXTRACT_DIR]),
+    AC_HELP_STRING([--with-extract-dir=EXTRACT_DIR],
+            [Build with the specified Extract library. By default we build with Extract automatically if <ghostpdl-directory>/extract exists.]
+            ),
     [
         EXTRACT_DIR="$withval"
         AS_IF([test x"$EXTRACT_DIR" != x"no"],
